src/Containers/Dropdown/Dropdown.tsx

```diff
@@ -22,6 +22,8 @@ export interface IDropdownProps extends React.HTMLAttributes<HTMLDivElement> {
     dropdownButton: React.ReactElement;
     dropdownWidth?: number;
     right?: boolean;
+    startState?: boolean;
```

Comment:
Why is this here?

Reply:
My bad, I didn't update my master branch before I cloned it. It should be fixed now.